app/ui/src/main/res/values/themes.xml

```diff
@@ -104,6 +104,8 @@
         <item name="openpgp_blue">#336699</item>
         <item name="openpgp_grey">#bbb</item>
         <item name="openpgp_dark_grey">#888</item>
+
+        <item name="selectableItemBackground">?android:attr/selectableItemBackground</item>
```

Comment:
The AppCompat theme already contains a `selectableItemBackground` attribute that does the right thing. Overwriting it will cause the blue Holo background to be used on older platform versions.

Reply:
sure